report/2.Analysis/Requirements.tex

```diff
@@ -38,4 +38,6 @@ \section{Requirements}
 By fulfilling the requirements specified in the \textit{must have} section, the device will be able to track a target with a rather predictable movement pattern and use the attached laser to indicate the tracking. 
 With the \textit{should have} requirements the functionality of the device is extended to track more advanced movement patterns, which in a real life scenario could be drones or birds in the air while still using the laser to indicate the tracking.
 If the \textit{could have} requirements are fulfilled the device will be able to calculate the distance to the observed target, calculate angle and power required to hit the target and fire a ballistic projectile at it.
-Finally, the \textit{won't have} requirements are deemed outside the scope of the current project and focus on the future application of the device, which would require more efficient projectiles in order to take down the observed target and being able to differentiate between whether the object is a threat or not.
\ No newline at end of file
+Finally, the \textit{won't have} requirements are deemed outside the scope of the current project and focus on the future application of the device, which would require more efficient projectiles in order to take down the observed target and being able to differentiate between whether the object is a threat or not.
+
+%TODO Transition from this to problem definition
```

Comment:
Add this transition when problem definition section is done.

Reply:
hvis vi har en section der er problem statement, så tænker jeg requirements kunne være en subsection dertil, som enten er før eller efter selve problemet